numba/tests/test_parfors.py

```diff
@@ -60,7 +139,9 @@ def null_comparer(a, b):
     """
     pass
 
-class TestParforsBase(TestCase):
+
+@needs_subprocess
+class TestParforsBase(TestCase, MemoryLeakMixin):
```

Comment:
```suggestion
class TestParforsBase(MemoryLeakMixin, TestCase):
```
mixin class goes first; earlier in the MRO